docs/content/preview/contribute/docs/docs-style.md

```diff
@@ -56,7 +58,54 @@ ysqlsh> CREATE TABLE $$__banking__$$;
 
 #### YSQL and YCQL code blocks
 
-Tag YSQL code blocks as `sql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.
+Tag YSQL code blocks as `plpgsql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.
```

Comment:
what's the reason for tagging YSQL code blocks as `plpgsql`?

Reply:
`plpgsql` has features like `tablespaces` and other postgres specific features that `sql` does not have . so they will be highlighted too.